src/web/hosts.rs

```diff
@@ -190,3 +216,31 @@ pub async fn release_all(
     service.free_all(&user.id().into()).await.unwrap();
     Redirect::to("/hosts")
 }
+
+#[derive(Deserialize)]
+pub struct GetHostsQuery {
+    login: String,
+}
+
+pub async fn get_leased_hosts_for_user(
```

Comment:
Maybe we can made it function for getting all hosts
and with query arguments for example `leased_by`, if we fill it we got hosts leased by this user